.github/workflows/apm-transport-stress-test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       DD_API_KEY: ${{ secrets.DD_SHARED_TESTS_API_KEY }}
       TRACER: python
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
can you move these bumps into their own PR?

Reply:
Done. Also, I'm not sure it really needs to happen anyway.